booking-engine-widget/getting-started.md

```diff
@@ -56,7 +56,8 @@ Place the following `<script>` code snippet in the `<head>` of your web page's H
 <iframe src="https://api.mews.com/distributor/aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"></iframe>
 ```
 
-> **Warning:** Please double-check that you've added the script as instructed and followed all the requirements above. If the script tag is not used correctly, it can cause unexpected problems even when it seems everything is working.
+
+> **Warning:** Please double-check that you've added the script as instructed and followed all the requirements above. If the script tag is not used correctly, it can cause unexpected problems even when it seems everything is working. See [Disable widget caching](./use-cases/disable-widget-caching.md)
```

Comment:
```suggestion
> **Warning:** Please double-check that you've added the script as instructed and followed all the requirements above. If the script tag is not used correctly, it can cause unexpected problems even when it seems everything is working. See [Disabling widget caching](./use-cases/disable-widget-caching.md)
```